packages/playwright-core/src/web/traceViewer/snapshotRenderer.ts

```diff
@@ -87,10 +87,10 @@ export class SnapshotRenderer {
     // Hide the document in order to prevent flickering. We will unhide once script has processed shadow.
     const hideAllStyle = '<style>*,*::before,*::after { visibility: hidden }</style>';
     const prefix = snapshot.doctype ? `<!DOCTYPE ${snapshot.doctype}>` + hideAllStyle : hideAllStyle;
-    html = prefix + html + `
-      <style>*[__playwright_target__="${this.snapshotName}"] { background-color: #6fa8dc7f; }</style>
-      <script>${snapshotScript()}</script>
-    `;
+    html = prefix + [
+      `<style>*[__playwright_target__="${this.snapshotName}"] { background-color: #6fa8dc7f; }</style>`,
+      `<script>${snapshotScript()}</script>`
+    ].join('') + html;
```

Comment:
Why this change?

Reply:
I see, thank you for the explanation. You can now merge hideAllStyle with this `<style>` tag - would be easier to follow.